frontend/src/scenes/data-management/definition/DefinitionView.tsx

```diff
@@ -149,7 +149,7 @@ export function DefinitionView(props: DefinitionLogicProps = {}): JSX.Element {
                                     </LemonButton>
                                 )}
 
-                                {hasTaxonomyFeatures && (
+                                {(hasTaxonomyFeatures || !isEvent) && (
```

Comment:
lack of context from my side: what are the different types of views that go through here? why is this an OR?

Reply:
so the view is shared between events and properties definitions

If you have taxonomy features paid for you can always go through... If it is a property (so is not an event 🤯) you can now go through because you will be able to edit one of three possible things.

I wonder if we've got an isProperty so the description can be less tortuous 🤔